packages/itwin/grouping-mapping-widget/src/widget/components/extension/ManualExtension.tsx

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { Button, LabeledTextarea, Text } from "@itwin/itwinui-react";
+import type { GroupExtensionProps } from "../../utils";
+import { LoadingSpinner } from "../utils";
+import "./ManualExtension.scss";
+
+const ManualExtension = ({ updateQuery, isUpdating, resetView }: GroupExtensionProps) => {
+  const [manualInput, setManualInput] = React.useState("");
+
+  return (
+    <div className="gmw-manual-form">
+      <Text>
+        Generate group by user defined ECSQL query. Please select
+        ECInstanceId column in the query.
```

Comment:
`Please select ECInstanceId and ECClassId columns in the query.`. This is a small thing, but it makes the extraction a bit more efficient.